src/eureka/S3_data_reduction/plots_s3.py

```diff
@@ -565,3 +568,307 @@ def median_frame(data, meta):
     plt.savefig(meta.outputdir+fname, dpi=300)
     if not meta.hide_plots:
         plt.pause(0.1)
+
+
+# Photometry
+def phot_lc(meta, data):
+    """
+    Plots the flux as a function of time.
+    """
```

Comment:
Please document the parameters for all these new functions - should just be a matter of copy-pasting things repeatedly

Reply:
Can you also put the figure number in the first line of the docstring like the other plots so it's easy to see from the docs pages which functions make which figures